src/components/badge/Badge.scss

```
@@ -2,46 +2,31 @@
 @import 'assets/styles/fonts.scss';
 @import 'assets/styles/common.scss';
 
-//지인 Badge color
-@mixin Beige() {
-  background-color: $orange-100;
-  color: $orange-500;
-}
-
-//동료 Badge color
-@mixin Purple() {
-  background-color: $purple-100;
-  color: $purple-500;
-}
-
-//가족 Badge color
-@mixin Green() {
-  background-color: $green-100;
-  color: $green-500;
-}
-
-//친구 Badge color
-@mixin Blue() {
-  background-color: $blue-100;
-  color: $blue-500;
-}
-
-.emoji-badge-container {
-  background-color: $color--black;
-  opacity: 54%;
-  border-radius: 32px;
-  width: 16px;
-  height: 21px;
-  padding: 8px 12px;
-}
-
 .text-badge-container {
   width: 41px;
   height: 20px;
   border-radius: 4px;
-  @include Flex(row, center, center, '');
-  @include Blue;
+  @include Flex(row, center, center, null);
+
+  &.BeigeBadge {
+    background-color: $orange-100;
+    color: $orange-500;
+  }
+  &.PurpleBadge {
+    background-color: $purple-100;
+    color: $purple-500;
+  }
+  &.GreenBadge {
+    background-color: $green-100;
+    color: $green-500;
+  }
+  &.BlueBadge {
+    background-color: $blue-100;
+    color: $blue-500;
+  }
 }
```

백그라운드랑 컬러 숫자들이 100, 500으로 동일해서 객체를 만들어서 사용하면 좋을거 같아용~!!!
```suggestion
$colors=(
  BaigeBadge: orange, // 제 실수로 인해 이름이 통일이 안되는데 수정할까용? ㅠㅠ 그게 낫겠죠  ......... ? 
  PurpleBadge: purple,
  GreenBadge: Green,
  BlueBadge: blue,
);

@include BadgeColor($class)  {
  background-color: `$${map-get($colors, $class)}-100`
  color: `$${map-get($colors, $class)}-500`
}
```
이런 느낌인데  `$${map-get($colors, $class)}-500` 이 부분은 테스트 안해보고 생각으로 짠거라 문자열 맞게(`$orange-500` -> 이렇게 들어가도록) 들어가게끔 수정해주시면 좋을거같아요! 
이러면 공통된 부분 줄일 수 있을거 같네용 ㅎㅎ